{{cookiecutter.project_dirname}}/{{cookiecutter.django_settings_dirname}}/settings.py

```diff
@@ -184,6 +184,8 @@ class ProjectDefault(Configuration):
 
     SESSION_COOKIE_SECURE = True
 
+    SESSION_COOKIE_SAMESITE = "Strict"
```

Comment:
This seems like a customization for a specific project, it doesn't seem necessary to me include it in the bootstrap.